fedora_elections/templates/base.html

```diff
@@ -9,62 +9,92 @@
         href="//fedoraproject.org/static/images/favicon.ico"/>
     <link rel='stylesheet' type='text/css'
         href="{{ url_for('static', filename='jquery-ui-1.9.2.custom.css') }}"/>
-    <link rel="stylesheet" type="text/css" media="screen"
-        href="{{ url_for('static', filename='koji.css') }}"/>
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.1/fedora-bootstrap.css"
+            type="text/css" rel="stylesheet" />
     <link rel="stylesheet" type="text/css" media="screen"
         href="{{ url_for('static', filename='elections.css') }}"/>
-    <link rel='stylesheet' type='text/css'
-         href='//fonts.googleapis.com/css?family=Comfortaa'/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
It would be ideal if we could get this stylesheet hosted in the Fedora infrastructure, so we don't depend on Google. Some users block Google URLs to prevent tracking. This is optional for this PR, since it was like this before anyway.